metaflow/plugins/events_decorator.py

```diff
@@ -219,12 +227,22 @@ class TriggerOnFinishDecorator(FlowDecorator):
 
     Parameters
     ----------
-    flow : Union[str, Dict[str, str]], optional
+    flow : Union[str, Dict[str, str]], optional, default: None
         Upstream flow dependency for this flow.
-    flows : List[Union[str, Dict[str, str]], optional
+    flows : List[Union[str, Dict[str, str]]], default: []
         Upstream flow dependencies for this flow.
-    options : dict, optional
+    options : Dict[str, Any], default: {}
         Backend-specific configuration for tuning eventing behavior.
+
+    MF Add To Current
+    -----------------
+    trigger -> metaflow.events.Trigger
```

Comment:
notice that both decorators can add the same thing. The definitions should be the same (one is just picked -- no checks). It will say that this applies to BOTH decorators.